### ISSUE TITLE
As a user I want a better frontend

### ISSUE THREAD

**Issue by hojott** [HY-OHTUPROJ-OSRM/osrm-project#144]
The frontend should have e.g. clearer navbar and body. It should also be more intuitive, and preferably easier on the eyes.

Maybe an easy mode and an advanced mode?

According to WCAG 2.2 standard?
